cli/config/configuration-qemu.toml.in

```diff
@@ -165,6 +165,11 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # result in memory pre allocation
 #enable_hugepages = true
 
+# Enable file based guest memory support. The default is an empty string which
```

Comment:
This is making the qemu config file deviate further from the firecracker one. Is this change qemu specific? If so, please could you mention that in the commit to make it clear.

Related: discussions on breaking up the config file into fragments - https://github.com/kata-containers/runtime/pull/1648

Reply:
Good point. Let me see if this at all will work with firecracker and then comment back here.

Reply:
@jodh-intel verified with @mcastelino that this feature will not work on firecracker. I have since removed the config from the firecracker hypervisor invocation and cleaned up the code. Will do a follow-on document update regarding this and hugepages.

Reply:
Thanks @ganeshmaharaj !